tests/end_to_end/test_shellshock_attack_detection/data/playbooks/configuration.yaml

```diff
@@ -0,0 +1,53 @@
+- name: Test setup
+  hosts: wazuh-manager
+  become: true
+  tasks:
+
+    - name: Check if Firewalld is installed on CentOS
+      shell: systemctl status firewalld --no-pager
+      register: firewall_check
+      when: ansible_facts['distribution'] == "CentOS"
+      # Ignore non-zero return codes for use in subsequent checks
+      ignore_errors: true
+
+    - name: Stop Firewalld if it's installed and active
+      shell: systemctl stop firewalld
+      # If the return code is 0, it means that firewalld is running, it is necessary to stop it to allow
+      # the shellshock attack.
+      when: (ansible_facts['distribution'] == "CentOS" and firewall_check.rc == 0)
+
+    - name: Check if Apache is installed or not on CentOS
+      shell: systemctl status httpd --no-pager
+      register: apache_check
+      when: ansible_facts['distribution'] == "CentOS"
+      # Ignore non-zero return codes for use in subsequent checks (Apache installation)
+      ignore_errors: true
+
+    - name: Install Apache Server on CentOS
+      shell: |
+        yum update httpd -y
+        yum install httpd -y
+      register: installation
+      # If the return code is 0, it means that Apache is running.
+      # If the return code is 3, it means that Apache is installed but not running.
+      when: (ansible_facts['distribution'] == "CentOS" and apache_check.rc != 0 and apache_check.rc != 3)
+
+    - name: Configure a localfile instance to collect the logs from Apache
+      blockinfile:
+        path: /var/ossec/etc/ossec.conf
+        marker: <!-- {mark} ANSIBLE MANAGED BLOCK -->
+        insertbefore: ^</ossec_config>
+        block: |
+          <localfile>
+          <log_format>apache</log_format>
+          <location>/var/log/httpd/access_log</location>
+          </localfile>
+
+    - name: Start Apache
+      shell: systemctl start httpd
+      # If the return code is 3, it means that Apache is not running.
+      # If Apache has been recently installed, it is necessary to start it
+      when: (apache_check.rc == 3 or installation is succeeded)
+
+    - name: Restart the manager
```

Comment:
This task should go before starting the apache service? For example, I would first group the configuration and startup tasks of the wazuh-manager so that it has time to start, while apache is being installed afterwards.

Reply:
@jmv74211 I don't think it is 100% necessary, because the task of restarting the Wazuh manager will not finish until the command does, but it is useful to prevent errors in the future. Again, when the manager is restarted via `systemd`, the command is supposed to terminate when the manager is finally up and running.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3028/commits/484e6016181d2288be201d4d6a4476f72ca7cbe1